rubix-bookkeeper/src/test/java/com/qubole/rubix/bookkeeper/TestBookKeeper.java

```diff
@@ -40,33 +38,29 @@
 {
   private static final Log log = LogFactory.getLog(TestBookKeeper.class);
 
-  private MetricRegistry metrics;
-  private Configuration conf;
+  private static final String TEST_CACHE_DIR_PREFIX = BookKeeperTestUtils.getTestCacheDirPrefix("bookKeeperTest");
```

Comment:
nit: change the directory name to `TestBookKeeper`

Reply:
@abhishekdas99 Addressed.